parts/windows/windowscsehelper.ps1

```diff
@@ -152,7 +217,7 @@ function Set-ExitCode
     )
     Write-Log "Set ExitCode to $ExitCode and exit. Error: $ErrorMessage"
     $global:ExitCode=$ExitCode
-    $global:ErrorMessage=$ErrorMessage
+    $global:ErrorMessage=($ErrorMessage -replace '\r*\n', '')

```

Comment:
Just a question: will this ErrorMessage be returned to the customer?

Reply:
It depends on the implementation in AKS RP. It seems like that only InternalOperationError is returned for now. We have a task to return these error messages to the users.